The agent version in the logs have SNAPSHOT

```
[opentelemetry.auto.trace 2020-12-10 16:51:13:549 +0000] [Attach Listener] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 0.5.1-SNAPSHOT
```

The `./gradlew printVersion` prints `-SNAPSHOT` when there is a tag e.g. `0.6.0` at HEAD and clean git status. 

I have also tried `./gradlew printVersion -Prelease` but that fails on `Cannot create a release version when there are uncommitted changes` even though the git status shows a clean tree.



```
./gradlew  printVersion                                                                                                                                                                                                       ploffay@Pavols-MBP

> Task :printVersion
0.5.2-SNAPSHOT

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1s
1 actionable task: 1 executed
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
~/projects/hypertrace/javaagent(main) » git tag 0.6.0                                                                                                                                                                                                                 ploffay@Pavols-MBP
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
~/projects/hypertrace/javaagent(main) » ./gradlew  printVersion                                                                                                                                                                                                       ploffay@Pavols-MBP

> Task :printVersion
0.6.1-SNAPSHOT

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1s
1 actionable task: 1 executed
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
~/projects/hypertrace/javaagent(main) » ./gradlew  printVersion -Prelease                                                                                                                                                                                             ploffay@Pavols-MBP

FAILURE: Build failed with an exception.

* What went wrong:
Cannot create a release version when there are uncommitted changes

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 646ms
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
~/projects/hypertrace/javaagent(main) » g s                                                                                                                                                                                                                       1 ↵ ploffay@Pavols-MBP
On branch main
nothing to commit, working tree clean
--------------------------------------
```

if the `./gradlew printVersion -Prelease` works we could add `-Prelease` here https://github.com/hypertrace/javaagent/blob/main/.circleci/config.yml#L72

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -9,7 +9,7 @@ pluginManagement {
 }
 
 plugins {
-    id("org.hypertrace.version-settings") version "0.1.5"
+    id("org.hypertrace.version-settings") version "0.1.6"
 }
 
 include(":javaagent")

